docs/configuration-reference/platforms/aws.md

```diff
@@ -268,4 +268,8 @@ To destroy the Lokomotive cluster, execute the following command:
 lokoctl cluster destroy --confirm
 ```
 
+## Logging
```

Comment:
Maybe we can put this in `docs/concepts/logging.md`? Then we can deduplicate and provide more information like what are our assumptions here.